### ISSUE TITLE
Show Base Content should be on by default

### ISSUE THREAD

**Issue by yuberee** [Facepunch/sbox-issues#7173]
### What can't you do?

In the assets browser, the Show Base Content option is off by default.
This confuses new developers and older ones alike.
I've seen too many people asking why the "Everything" option doesn't show everything, and it looks like it will always be asked.

### How would you like it to work?

Show Base Content on by default

### What have you tried?

.

**Comment by Gmod4phun** [comment]
I don't think this should even be a toggleable option, and if so, make it very clear that you can show/hide it using a toggle that is always visible, not by right clicking on the empty space in the content tree list and suddenly realizing you can do that.

**Comment by yuberee** [comment]
> I don't think this should even be a toggleable option, and if so, make it very clear that you can show/hide it using a toggle that is always visible, not by right clicking on the empty space in the content tree list and suddenly realizing you can do that.

Personally none of our games use base content and I can see many others in the future, but I know we're just built different and we're the exception
